### ISSUE TITLE
--editorconfig does not work with --stdin

### ISSUE THREAD

**Issue by lwuk** [pinterest/ktlint#869]
## Expected Behavior
The `.editorconfig` file passed to `--editorconfig` should have its configuration used by ktlint.

## Observed Behavior
`--editorconfig` argument's configuration is not used when `--stdin` option is used.

## Steps to Reproduce
Make a kotlin file at `~/src/SomeKotlinFile.kt` and an editor config at `~/config/.editorconfig`. 

`cat <path-to-home>/src/SomeKotlinFile.kt | ktlint --stdin --editorconfig=<path-to-home>/config/.editorconfig` won't use the config's options, while `ktlint <path-to-home> --editorconfig=<path-to-home>/config/.editorconfig` will.

It will use an .editorconfig in the current directory instead, so the config **will** be taken into account if `~/config/` is the working directory.

## Your Environment

* Version of ktlint used: 0.38.1
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line

**Comment by Tapchicoma** [comment]
Yes, usage of `--stdin` overrides `--editorconfig` option. Workaround would be:
```sh
cd ~/config/ && cat ~/src/SomeKotlinFile.kt | ktlint --stdin
```
